python/lsst/summit/utils/tmaUtils.py

```diff
@@ -1780,12 +1788,14 @@ def printTmaDetailedState(tma: TMAStateMachine) -> None:
         #   azimuth - Power:          ON Motion:               STOPPED InPosition: True        # noqa: W505
         # elevation - Power:          ON Motion: MOVING_POINT_TO_POINT InPosition: False       # noqa: W505
         for axis in axes:
-            print(
-                f"{axis:>{axisPad}} - "
-                f"Power: {p[f'{axis}SystemState'].name:>{powerPad}} "
-                f"Motion: {p[f'{axis}MotionState'].name:>{motionPad}} "
-                f"InPosition: {p[f'{axis}InPosition']}"
-            )
+            f"{axis:>{axisPad}} - "
+            power_state = p[f"{axis}SystemState"]
+            motion_state = p[f"{axis}MotionState"]
+            if isinstance(power_state, PowerState):
+                f"Power: {power_state.name:>{powerPad}} "
+            if isinstance(motion_state, AxisMotionState):
+                f"Motion: {motion_state.name:>{motionPad}} "
+            f"InPosition: {p[f'{axis}InPosition']}"
```

Comment:
Two things:
1) please don't introduce snake_case to camelCase files
2) much more importantly, this totally changes the behaviour of the code!

- You have a floating f-string with no print statement at the start of the change, so that doesn't print.
- You've split one print statement into several, without setting `end=''` so this now prints on many lines where before it was a single line
- You finish with another unprinted f-string.

So I'd either: a) unpack everything, put in whatever asserts you need to to make mypy happy, and then call my print statement, or do it the way you're doing it, but put the missing print statements back in, and add `end=''` to all of them, and I _think_ that will give you the old behaviour back. In general, please be careful not to change the behaviour of things, especially things which are basically impossible to have decent tests for.

Reply:
Before:
<img width="807" alt="image" src="https://github.com/lsst-sitcom/summit_utils/assets/18741538/a783b9b5-7008-484a-8506-ac75dae960ae">
After:
<img width="810" alt="image" src="https://github.com/lsst-sitcom/summit_utils/assets/18741538/9b1d5e60-80a4-4997-8cc1-1d64f9d68a13">

Reply:
Yes, I forgot the print statements... Sorry! And snake_case...

Reply:
OK, you still haven't fixed this by the looks of things. Please let me know when you have and I'll re-re-review it.